wordpress/mysql/Dockerfile_LOCAL

```diff
@@ -0,0 +1,34 @@
+# =============================================================================
+# mysql:5.7
+#
+# Modification: WordPress MySQL error log Fail2Ban setup
+#
+# =============================================================================
+FROM mysql:5.7
+LABEL maintainer="Jacob Thompson <jacob.thompson@wiwasolvet.ca>"
+
+RUN apt-get update &&  \
+    apt-get install -y \
+    nano \
+    vim
+
+# Setup mysql auth log for Fail2Ban regex
+#RUN touch /var/log/mysql/error.log && \
+#    printf "# This is the MySQL login authorization log." >> /var/log/mysql/error.log
+# Note: must create log folder and error.log file on first run, otherwise container will restart continually.
+
+RUN printf "%s\n" "[mysqld]" \
+	"log_warnings = 2" \
+	"log_timestamps = SYSTEM" \
+	"log_error = /var/log/mysql/error.log" >> /etc/mysql/my.cnf
```

Comment:
logrotated almost worked, it made a new file in ckanlog.log but CKAN's user kept writing to old file, this will become an issue eventually, either for the log software we use to read the files, or in other scenarios. This is true for all mounted volume logs from our docker containers.